lib/bin/bt_ll_pcft/include/ble_hci_vsc.h

```diff
@@ -4,6 +4,13 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+/**
+ * @file ble_hci_vsc.h
+ *
+ * @defgroup ble_hci_vsc_api LE Audio Controller for nRF5340 API
+ * @{
+ */
+
 #ifndef _BLE_HCI_VSC_H_
 #define _BLE_HCI_VSC_H_
 
```

Comment:
Before I review this header, I see that it is incomplete, as the defines, structs and enums in lines 19-111 are not documented. 

@koffes , can you add API documentation for these elements?